src/library/scala/collection/mutable/ArrayBuffer.scala

```diff
@@ -150,6 +150,12 @@ class ArrayBuffer[A] private (initElems: Array[AnyRef], initSize: Int)
       throw new IllegalArgumentException("removing negative number of elements: " + count)
     }
 
+  @deprecated("ArrayBuffer no longer extends Builder[ArrayBuffer].", "2.13.0")
```

Comment:
`Builder[ArrayBuffer]` is not a proper type. If we want to be accurate we should write:

> `ArrayBuffer[A]` no longer extends `Builder[A, ArrayBuffer[A]]`

Reply:
Oh yes, thank you